src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="kalendar kalendar__start-week--sun kalendar__days-in-month--29">
```

Comment:
typo
```suggestion
    <div class="calendar calendar__start-week--sun calendar__days-in-month--29">
```
fix all such cases